t/t2290-job-update.t

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
+test_description='Test flux update command'
+
+. $(dirname $0)/sharness.sh
+
+if flux job submit --help 2>&1 | grep -q sign-type; then
+	test_set_prereq HAVE_FLUX_SECURITY
+fi
+
+test_under_flux 1 job
+
+flux setattr log-stderr-level 1
+
+runas_guest() {
+	local userid=$(($(id -u)+1))
+        FLUX_HANDLE_USERID=$userid FLUX_HANDLE_ROLEMASK=0x2 "$@"
+}
+
+test_expect_success 'flux update requires jobid and keyval args' '
+	test_expect_code 2 flux update 1234 &&
+	test_expect_code 2 flux update
+'
+test_expect_success 'submit jobs for testing' '
+	inactive_jobid=$(flux submit --wait-event=clean true) &&
+	jobid=$(flux submit --urgency=hold sleep 60)
+'
+test_expect_success 'invalid jobid fails' '
+	test_expect_code 1 flux update f123 duration=10m 2>invalid-jobid.err &&
+	test_debug "cat invalid-jobid.err" &&
+	grep "unknown job id" invalid-jobid.err
+'
+test_expect_success 'update to inactive job fails' '
+	test_expect_code 1 flux update $inactive_jobid duration=1m \
+		2>inactive-jobid.err &&
+	test_debug "cat inactive-jobid.err" &&
+	grep "job is inactive" inactive-jobid.err
+'
+test_expect_success 'invalid jobspec key cannot be updated' '
+	test_expect_code 1 flux update $jobid foo=bar
+'
+test_expect_success 'invalid jobspec key cannot be updated' '
+	test_expect_code 1 flux update $jobid foo=bar
+'
```

Comment:
duplicates :-)

Reply:
Thanks! Just as a warning, I'm still working on this WIP PR ;-) (though honestly I probably would have missed this duplicate, so good catch) I just wanted to see the coverage of the current set of tests -- though I don't see a report here so :-(